languages/es.json

```
@@ -160,6 +163,9 @@
   "building-workshops-info3C": null,
   "host-header": "Organizando un evento NodeSchool",
   "host-info": "NodeSchool es un esfuerzo impulsado completamente por la comunidad y es 100% open source. Esto significa que cualquiera puede utilizar los materiales de NodeSchool y/o organizar un evento.",
+  "host-info-cost": "La única expectativa para los organizadores es que los eventos sean sin fines de lucro(gratuitos o de bajo costo) y abiertos para la administración de eventos (para que nadie pueda \"controlar\" los eventos).",
```

We're repeating the word "eventos" too much. Maybe something like:
```suggestion
  "host-info-cost": "Lo único que se espera de los organizadores es que los eventos sean sin fines de lucro (gratuitos o de bajo costo) y que la administración tenga un modelo abierto (de esa manera nadie puede \"controlar\" los eventos).",
```